MinimedKit/PumpEvents/PumpEvent.swift

```diff
@@ -21,3 +21,26 @@ public protocol PumpEvent : DictionaryRepresentable {
     }
     
 }
+
+public extension PumpEvent {
+    public func isDelayedAppend(withPumpModel pumpModel: PumpModel) -> Bool {
+        
+        if !pumpModel.mayHaveOutOfOrderEvents {
```

Comment:
Boluses (non-square) on both pump models may be out of order, so this method should be removed.